apps/mobile/src/hooks/services/useSafeOverviewService.ts

```diff
@@ -5,19 +5,24 @@ import { useMemo } from 'react'
 import { makeSafeId } from '@/src/utils/formatters'
 import { useSafesGetOverviewForManyQuery } from '@safe-global/store/gateway/safes'
 
-export const useSafeOverviewService = (safeAddress: string) => {
+export const useSafeOverviewService = (safeAddress?: string) => {
   const chainIds = useAppSelector(selectAllChainsIds)
   const safes = useMemo(
-    () => chainIds.map((chainId: string) => makeSafeId(chainId, safeAddress)),
+    () => (safeAddress ? chainIds.map((chainId: string) => makeSafeId(chainId, safeAddress)) : []),
```

Comment:
it was necessary because since we fetch the safe information in the beginning of the app, the address might not exists and hooks can not be conditional, so there were only 2 possible options, either create a wrapper component and put it under a render condition or either make this type optional.

I've decided to do it like that because if I create a wrapper component and put it under a a render condition, the whole app tree would re-render when the active safe changes (because the render component would trigger that), and to avoid that we would need to wrap the children under an useMemo and so on, which does not makes sense IMO.